editor/src/utils/value-parser-utils.ts

```diff
@@ -267,6 +267,14 @@ export function parseBoolean(value: unknown): ParseResult<boolean> {
   }
 }
 
+export function parseFalse(value: unknown): ParseResult<false> {
+  if (typeof value === 'boolean' && value === false) {
```

Comment:
`&& value === false`
This feels so very wrong general, and yet in this specific case it feels like the correct thing to do. I am so conflicted right now